webapp/src/domain/item.ts

```diff
@@ -0,0 +1,12 @@
+export interface Item {
```

Comment:
Have both `Item` and `VotingItem` is confusing, an better option would be to convert the response to our `VotingItem` upon receiving the data

Reply:
Yes, I agree. Thank you for the correction!